go.mod

```diff
@@ -23,6 +23,7 @@ require (
 )
 
 require (
+	github.com/argoproj/pkg v0.13.7-0.20230626144333-d56162821bd1
```

Comment:
why do we need this dependency?

Reply:
I'm using it [here](https://github.com/numaproj/numaplane/pull/156/files#diff-39d087e5d3b033cf14587cb8e05de77a3842c876d18e9d3790ae5c3493b4a5dfR15) if we don't wanted to use then need to copy that fn here with their dependencies here.